.github/workflows/test.yml

```diff
@@ -11,16 +11,19 @@ on:
       - '**.py'
 
 jobs:
-  test-cpython:
-    name: Test CPython ${{ matrix.python_version }} on ${{ matrix.os }}
+  test:
     runs-on: ${{ matrix.os }}
-
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
         # Python 3.4 is not available from actions/setup-python@v1.
-        python_version: ['2.7', '3.5', '3.6', '3.7', '3.8']
+        python_version: ['2.7', '3.5', '3.6', '3.7', '3.8', 'pypy2', 'pypy3']
```

Comment:
Beware of the bugs with PyPy under macOS: https://github.com/actions/setup-python/issues/56.

Reply:
It looks like we're insulated from this, by nox's interpreter discovery logic.